astropy/coordinates/baseframe.py

```diff
@@ -156,6 +156,69 @@ def __new__(mcls, name, bases, members):
                 'Could not find all expected BaseCoordinateFrame class '
                 'attributes.  Are you mis-using FrameMeta?')
 
+        # Unless overridden, velocity name defaults are:
+        #   * pm_{lon}_cos{lat} for SphericalCosLatDifferential
+        #   * radial_velocity for any d_distance, including RadialDifferential
+        #   * v_{x,y,z} for d_{x,y,z} for CartesianDifferential
+        # where {lon} and {lat} are the names of the angular components
+        if repr_info is None:
+            repr_info = dict()
+
+        if r.SphericalRepresentation in repr_info:
+            sph_mappings = repr_info[r.SphericalRepresentation]
+            component_map = dict([(m.reprname, m.framename)
```

Comment:
Total nitpick, but it seems clearer to do:
```
component_map  = {m.reprname: m.framename for m in sph_mappings}
```

I'd similarly replace `dict()` with `{}` as I think that is more standard (but this really is nitpicking).